CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- An issue where both the ASB and the CLI point to the same default directory `xmr-btc-swap` for storing data.
+  The asb now uses `xmr-btc-swap-asb` and the CLI `xmr-btc-swap-cli` as default directory.
+  This is a breaking change.
+  If you want to access data created by a previous version you will have to rename the data folder or one of the following.
+  For the CLI you can use `--data-dir` to point to the old directory.
+  For the ASB you can change the data-dir in the config file of the ASB.
```

Comment:
You probably wanted this to be an enumeration I guess? (Untested what this renders to)

```suggestion
  If you want to access data created by a previous version you will have to rename the data folder or one of the following:
  1. For the CLI you can use `--data-dir` to point to the old directory.
  2. For the ASB you can change the data-dir in the config file of the ASB.
```

Reply:
`dprint` not complaining locally - CLion renders it like this:

![image](https://user-images.githubusercontent.com/5557790/116636075-c84a2d80-a9a3-11eb-8f27-a0dc590fbb22.png)